app/src/main/java/org/mozilla/fenix/Config.kt

```diff
@@ -40,6 +40,12 @@ enum class ReleaseChannel {
             else -> false
         }
 
+    val isBeta: Boolean
+        get() = when (this) {
+            FenixBeta -> true
```

Comment:
Should we add `FennecBeta` too?

👀 @pocmo can take it out if he doesn't like it

Reply:
Done